src/pass/llvmgen.cpp

```diff
@@ -3,6 +3,7 @@
 #include "llvm/IR/DataLayout.h"
 #include "llvm/IR/Function.h"
 #include "llvm/IR/InstrTypes.h"
+#include "llvm/IR/IntrinsicsNVPTX.h"
```

Comment:
why do we need this in this PR?

Reply:
It's needed for the atomic operations intrinsics. But I realize I should remove the existing import below, which is behind the ENABLE_CUDA flags. I have fixed that.